test/org/sagebionetworks/bridge/services/SubstudyServiceTest.java

```diff
@@ -60,6 +61,16 @@ public void getSubstudy() {
         verify(substudyDao).getSubstudy(TestConstants.TEST_STUDY, "id");
     }
     
+    @Test
+    public void getSubstudyIds() {
+        SUBSTUDIES.get(0).setId("substudyA");
+        SUBSTUDIES.get(1).setId("substudyB");
```

Comment:
We shouldn't be modifying constants. If these are truly constants, they should be statically initialized and then never changed. If these aren't constants, we should create them in before() to ensure they're being reset with each test.

Reply:
Created new list of subsidies for this test.